src/utils/webdriverutils/index.js

```diff
@@ -0,0 +1,13 @@
+const { expect } = require('chai');
+
+function assertScreenMatches(screenshots) {
+  var mismatchPercentage = browser.options.visualRegression.compare.misMatchTolerance;
+
+  screenshots.forEach((screenshot) => {
+      expect(screenshot.misMatchPercentage, `Mismatch is too high ${screenshot.misMatchPercentage}`).to.be.lessThan(mismatchPercentage);
+  }); 
+}
+
+module.exports = {
+  assertScreenMatches,
+};
```

Comment:
I am still curious to see if we can patch Jest's `expect` to run these—having mocha/chia and jest tests in the same directory kind of bugs me. Just thinking 🤔

Reply:
https://jestjs.io/docs/en/expect.html#expectextendmatchers

Reply:
Done. https://github.com/cremalab/crema-app-web/pull/17/commits/497b2906bd047fdbeb54be39c3f60e30e0d7b7f9